actionpack/lib/action_controller/renderer.rb

```diff
@@ -20,7 +20,7 @@ module ActionController
   # #render method allows you to use any options as when rendering in controller.
   # For example,
   #
-  #   FooController.render :action, locals: { ... }, assigns: { ... }
+  #   FooController.render :index, locals: { ... }, assigns: { ... }
```

Comment:
@schneems On a second thought, I think this line is fine for API documentation as it is. May be we can change `:action` to `:index`, but without that also this is fine. We are not giving too much details anyway in this example about `locals` and `assigns` too.

But I think we should add some documentation about how to render arbitrary template in guides? I couldn't find anything as of now.